test/functional/page_objects/getting_started_page.js

```diff
@@ -1,13 +1,30 @@
-export function GettingStartedPageProvider({ getService }) {
+export function GettingStartedPageProvider({ getService, getPageObjects }) {
 
   const log = getService('log');
   const testSubjects = getService('testSubjects');
 
+  const PageObjects = getPageObjects(['common']);
+
   class GettingStartedPage {
```

Comment:
Why is this even an instantiated class? I'm not seeing any local state or anything. All the methods just call static methods, or instance methods off already instantiated objects.